File: week-08/day-4/todos_mysql/src/main/java/com/greenfox/tgabor/todos_mysql/model/entity/Assignee.java
package com.greenfox.tgabor.todos_mysql.model.entity;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.util.List;

@Entity
@Getter
@Setter
@NoArgsConstructor
public class Assignee {
  @Id
  @GeneratedValue
  private Long id;
  private String name;
  private String email;
  @OneToMany(cascade = {CascadeType.PERSIST, CascadeType.MERGE})
  @JoinColumn(name="assignee_id")
  private List<Todo> todoList;

  public Assignee(String name, String email) {
    this.name = name;
    this.email = email;
  }
}

File: week-11/day-4/TodoApp/TodoApp/Models/Assignee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TodoApp.Models
{
    public class Assignee
    {
        public long AssigneeId { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public List<Todo> TodoList { get; set; }
    }
}
